fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -154,7 +154,7 @@ def additional_upload_parameters
       # As there was no communication from Apple, we don't know if this is a temporary
       # server outage, or something they changed without giving a heads-up
       if ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"].to_s.length == 0
-        ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t DAV"
+        ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t DAV,Signiant"
```

Comment:
Hey @silvervest 👋 

Thanks for creating the PR and adding a detailed comment about it https://github.com/fastlane/fastlane/issues/15375#issuecomment-536396594. 

I was also testing the changes from https://github.com/fastlane/fastlane/pull/15379 and noticed the same behaviour - it took ~45 minutes before it accepted DAV as the transport of choice (most time spent on Signiant and Aspera failures). 

Considering above and https://github.com/fastlane/fastlane/issues/15375#issuecomment-536057491, I would give a go this PR for the following reasons: 

* `iTMSTransporter`'s auto-updates (introduced in 02/16/10)
* for any backward compatibility issues (which should happen hardly ever), users will be able to simply override `DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS` in theirs `Fastlfile`, i.e. `ENV['DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS'] = -t Dav -t Signiant`
* checking the `iTMSTransporter` version takes ~6 seconds (pretty slow)

Cheers 🙇